src/test/scala/com/databricks/spark/avro/AvroSuite.scala

```diff
@@ -42,6 +42,43 @@ private[avro] object TestUtils {
    * copied from Spark with minor modifications made to it. See original source at:
    * github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/util/Utils.scala
    */
+
+  def checkReloadMatchesSaved(testFile: String, avroDir: String) = {
```

Comment:
The comment above this function (This function deletes a file or a directory...) is for `deleteRecursively`, please move it there.